index.js

```diff
@@ -1687,7 +1742,58 @@
   //. //
   //. //   The value at position 1 is not a member of ‘{ x :: FiniteNumber, y :: FiniteNumber }’.
   //. ```
-  function RecordType(fields) {
+  //.
+  //. ##### Extending record types
+  //.
+  //. By using the `parent` argument of the RecordType constructor, one can
```

Comment:
Let's replace `RecordType` with `` `RecordType` `` or `` [`RecordType`][] `` (depending on whether you think it should be a link).